Source/DafnyCore/Compilers/Python/Compiler-python.cs

```diff
@@ -1574,10 +1574,13 @@ private class ClassWriter : IClassWriter {
     protected override void TrStmtList(List<Statement> stmts, ConcreteSyntaxTree writer) {
       Contract.Requires(cce.NonNullElements(stmts));
       Contract.Requires(writer != null);
-      if (stmts.All(s => s.IsGhost)) {
+      var listWriter = new ConcreteSyntaxTree();
+      base.TrStmtList(stmts, listWriter);
+      if (listWriter.Descendants.OfType<LineSegment>().Any()) {
```

Comment:
Maybe we could add something like `IsEmpty`.

Reply:
This implementation isn't perfect so I'd rather not hide it behind something that looks better.

Reply:
If it's not, we could just amend it existing check.

Reply:
Quite frankly I'm tempted to just always print the `pass`.

Reply:
Since this is slightly better and we already wrote the code, shall we just leave it?